docs/how-to-docs/drone-ci.md

```diff
@@ -0,0 +1,385 @@
+## Drone CI (v1)
+
+- [Drone CI](#drone-ci)
+  - [Overview](#overview)
+  - [Repository Migration](#repository-migration)
+  - [Example Pipelines](#example-pipelines)
+  - [Services](#services)
+    - [Docker-in-Docker](#docker-in-docker)
+    - [Anchore Image Scanning](#anchore-image-scanning)
+    - [ECR](#ecr)
+
+### Overview
+
+Drone CI v1 is now available within ACP. For a short time, we will be running both old and new instances to allow projects to migrate each of their own repositories over to the new instances. For reference, the endpoints are:
+
+- [OLD-GITLAB] https://drone-gitlab.acp.homeoffice.gov.uk
+- [OLD-GITHUB] https://drone.acp.homeoffice.gov.uk
+- [NEW-GITLAB] https://drone-gl.acp.homeoffice.gov.uk
+- [NEW-GITHUB] https://drone-gh.acp.homeoffice.gov.uk
+
+**Notable changes:**
+
+- Builds are now Kubernetes Native: Drone CI will create ephemeral Kubernetes Namespaces and Pods for your pipelines, which are torn down entirely after completion. This enables complete isolation of builds and a greater level of security.
+- No long-running agents: As with above, there is no longer a requirement for long running agents to run Drone pipeline steps within. This helps keep costs down significantly as the CI environment can run at a lower capacity by default and scale automatically when required.
+- Cron Scheduling: Drone CI now supports the use of [cron jobs](https://docs.drone.io/cron/) to execute pipelines on time-based schedules.
+- Parallelism: Multiple Pipeline objects can be defined for a single repository and executed in parallel. In addition, build steps can be described as a [directed acyclic graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph), allowing for more complex execution flows.
+- Organisation Secrets: Secrets can be created and managed at an Organisation level, i.e. made available to all repositories within your Gitlab group.
+- Container Registry Authentication: Pipeline steps referencing images stored in ACP ECR or Artifactory can be pulled by default, without the need to manually authenticate with separate credentials.
+
```

Comment:
Add a note here for Starlark and any other recent changes or plugins that were introduced?

Reply:
Note on starlark added